src/modules/notes/controllers/createNote.js

```diff
@@ -1,11 +1,22 @@
 import { HttpStatusCode } from '#src/constants';
 import {CustomAPIError} from '#errors/custom-error';
 import { PROVIDE_VALUES } from '../constants/index.js';
+import { schemas } from '../schemas.js';
 
 export const createNote = (req, res) => {
-	const noParameters = !Object.keys(req.body).length;
+	const { body } = req;
+	const noParameters = !Object.keys(body).length;
 	if (noParameters) {
 		throw new CustomAPIError(HttpStatusCode.BAD_REQUEST, PROVIDE_VALUES);
 	}
+	
+	const result = schemas.noteCreate.validate(body);
+	const { error } = result;
```

Comment:
There is no critical need to separate variables in such details way:
```suggestion
	const { error } = schemas.noteCreate.validate(body);
```

Reply:
done